webapp/portlet/src/main/webapp/vue-apps/user-setting-mobile/components/MobileSettingsDownloadWindow.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+    <v-card 
+    class="ma-4 w-80"
+    flat>
+    <v-toolbar
+      class="border-box-sizing"
+      flat>
+      <v-btn
+        class="mx-1"
+        icon
+        height="36"
+        width="36"
+        @click="$emit('back')">
+        <v-icon size="20">
+          {{ $vuetify.rtl && 'mdi-arrow-right' || 'mdi-arrow-left' }}
+        </v-icon>
+      </v-btn>
+      <v-toolbar-title class="ps-0">
+        {{ $t('MobileSettings.download') }}
+      </v-toolbar-title>
+      <v-spacer />
+    </v-toolbar>
+    <span v-if="!isMobile" class="d-flex mx-16 my-6 justify-space-between flex-wrap align-center text-left">
+            <v-responsive max-width="70%">
+                <span class="body-1" >
+                    {{ $t('MobileSettings.download.detail') }}
+                </span>
+            </v-responsive>
+            <span   >
+                <span class="background-qrcode-image h1">
+                     <span
+                        class="mt-2"
+                        height="145"
+                        width="145"
+                        id="qrcode">
+                    </span>
+                </span>
+            </span>
+              <v-container>
+                <v-row align="center">
+                    <v-divider></v-divider><span class="grey--text text--lighten-2">{{ $t('MobileSettings.download.or') }}</span><v-divider></v-divider>
+                </v-row>
+              </v-container>
+          </span>
+    <span class="d-flex justify-space-between flex-wrap align-center text-left" 
+          :class="`${isMobile ? 'mt-10' : ''}`" >
+        <div class="ml-16 mt-n9">
+          <span >
+              <span class="body-1 ">
+                  {{ $t(getAppropriateStringName()) }}
+              </span>
+          </span> 
+        </div>  
+          <div class="mb-10 mr-5">
+            <div v-if="getMobileOperatingSystem()==='android'||getMobileOperatingSystem()==='unknown'" class="d-block-flex">
+                <a class= "mr-16 mb-6  d-block" :class="`${isMobile ? 'ml-14' : ''}`"
+                href='https://play.google.com/store/apps/details?id=org.exoplatform&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1' 
```

Comment:
This URL is sufficient no ?
```suggestion
                href='https://play.google.com/store/apps/details?id=org.exoplatform' 
```